ATL/schema.sql

```diff
@@ -0,0 +1,60 @@
+-- Create Passengers table
+CREATE TABLE Passengers (
+    PassengerID INT IDENTITY(1,1) PRIMARY KEY,
+    FirstName NVARCHAR(50) NOT NULL,
+    LastName NVARCHAR(50) NOT NULL,
+    Age INT

```

Comment:
Storing age like this is a really bad idea. Try to think about why that is, we'll discuss it during the class.